label_studio/io_storages/gcs/models.py

```diff
@@ -180,28 +180,17 @@ def iterkeys(self):
             return_key=True,
         )
 
-    def get_data(self, key) -> Union[dict, list[dict]]:
+    def get_data(self, key) -> tuple[list[dict], list[int | None], list[int | None]]:
         if self.use_blob_urls:
-            return {settings.DATA_UNDEFINED_NAME: GCS.get_uri(self.bucket, key)}
-        data = GCS.read_file(
+            task = {settings.DATA_UNDEFINED_NAME: GCS.get_uri(self.bucket, key)}
+            return [task], [None], [None]
+        blob_str = GCS.read_file(
             client=self.get_client(),
             bucket_name=self.bucket,
             key=key,
-            convert_to=GCS.ConvertBlobTo.JSON,
+            convert_to=GCS.ConvertBlobTo.NOTHING,
```

Comment:
do we have to pass `convert_to` at all in this case?

Reply:
Nope! Just tried to be very explicit. But it's not needed, will remove it